src/cljs/main/broadfcui/page/workspaces_list.cljs

```diff
@@ -21,23 +21,26 @@
 
 (def row-height-px 56)
 
-(def disabled-text
-  (str "This workspace contains controlled access data. You can only access the contents of this workspace if you are "
-       "dbGaP authorized for TCGA data and have linked your FireCloud account to your eRA Commons account."))
+(defn- disabled-text [protected?]
```

Comment:
I agree. In fact, I recently recognized that part of the challenge of reading cljs is knowing whether or not to trust what's a noun and what's a verb.

I couldn't think of a better verb than something like "get", "determine", "calculate", etc. and I didn't like any of those. Since the goal is to have the text in a single place and since there are only 2 possibilities, I just made 2 `def`s instead of a boolean `defn`.